src/app/Routes.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import {
+  BrowserRouter as Router,
+  Route,
+  Switch,
+  Redirect
+} from 'react-router-dom'
+
+import { FireWeatherPage } from 'features/fireWeather/FireWeatherPage'
+import { FWICalculatorPageWithDisclaimer } from 'features/fwiCalculator/FWICalculatorPageWithDisclaimer'
+
+const NoMatch = () => <div>Page not found.</div>
+
+const shouldShowDisclaimer = process.env.REACT_APP_HIDE_DISCLAIMER === undefined
+
+export const Routes = () => {
+  return (
+    <Router>
+      <Switch>
+        <Redirect exact from="/" to="/percentile-calculator" />
+
+        <Route path="/percentile-calculator">
+          <FWICalculatorPageWithDisclaimer
+            showDisclaimer={shouldShowDisclaimer}
+          />
+        </Route>
+
+        <Route path="/fire-weather">
+          <FireWeatherPage />
+        </Route>
+
+        <Route>
+          <NoMatch />
```

Comment:
does this throw a 404?

Reply:
Nope it renders `<div>Page not found.</div>`

Reply:
but shouldn't it return a 404 if the page doesn't exist?

Reply:
That would be a weird 404, because technically, from server perspective, the page has been found. We probably need a ticket for a "page not found" page - which isn't quite the same as a 404 page.

Reply:
You are suggesting we need to send status code 404 instead of 200 with not found page?

Reply:
Fair enough

Reply:
This is like "react couldn't find the page you want" - which is not a 404? Our 404 could point to the same "page not found" page?